dim/lib/dim/exporter/templates/raw_rst/requirement.rst.erb

```diff
@@ -0,0 +1,13 @@
+.. _`dim-req-<%= requirement_id %>`:
+
+<%= requirement_id %>
+<%= "-" * requirement_id.length %>
+
+.. list-table::
+
+   * - **<%= requirement_id %>** | <%= requirement.review_status %> | <%= [asil, requirement.origin].compact.join(" | ") %>
+   * - **Tags:** <%= requirement.tags.join(', ') %>
+   * - **Test Setups:** <%= requirement.test_setups.join(', ') %>
```

Comment:
In newer versions, we call it "Verification Methods".

Reply:
Changed as suggested.